R/Connection.R

```diff
@@ -1,6 +1,30 @@
 #' @include Driver.R
 NULL
 
+#' Supported Connection Attributes
+#'
+#' These (pre) connection attributes are supported and can be passed as
+#' part of the `dbConnect` call in the named list `attributes` parameter:
+#'
+#' * azure_token: This should be a string scalar; in particular Azure Active
+#'   Directory authentication token.  Only for use with Microsoft SQL Server and
+#'   with limited support away from the OEM Microsoft driver.
+#' @rdname ConnectionAttributes
+#' @aliases ConnectionAttributes
+#' @usage NULL
+#' @format NULL
+#' @examples
+#' \dontrun{
+#' conn <- dbConnect(
+#'   odbc::odbc(),
+#'   dsn = "my_azure_mssql_db",
+#'   Encrypt = "yes",
+#'   attributes = list("azure_token" = .token)
+#' }
+#' @export
+SUPPORTED_CONNECTION_ATTRIBUTES <-
```

Comment:
Do you want to export this object, or just make this a home for documenting any additional attributes?

Reply:
Good eye - un-exported.

Thanks for all the feedback.